### PR TITLE
Joomla 4.0 updated to PHP 7.2.5 as minimum

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue https://volunteers.joomla.org/departments/production/reports/1128-production-dept-meeting-november-05-2019

### Summary of Changes

As of https://volunteers.joomla.org/departments/production/reports/1128-production-dept-meeting-november-05-2019 the minimum PHP for Joomla 4.0 is 7.2.5

### Testing Instructions

Install on php lower than 7.2.5 -> see the message
install on php above 7.2.5 -> (and meet the other requirements) see no message

### Expected result

php 7.2.5 is minimum

### Actual result

php 7.2.0 is minimum

### Documentation Changes Required

none as this is a 4.0 change to be documented with the 4.0 PR that will follow

### CODE REVIEW

File: administrator/components/com_admin/postinstall/joomla40checks.php [hunk 1]
```diff
@@ -70,6 +70,6 @@ function admin_postinstall_joomla40checks_condition()
 		return true;
 	}
 
-	// PHP minimum version is 7.2
-	return version_compare(PHP_VERSION, '7.2', 'lt');
+	// PHP minimum version is 7.2.5 (as of https://volunteers.joomla.org/departments/production/reports/1128-production-dept-meeting-november-05-2019)
```
Comment (brianteeman) [inline administrator/components/com_admin/postinstall/joomla40checks.php]: I really dont think we need to add the link

Comment (zero-24) [reply administrator/components/com_admin/postinstall/joomla40checks.php]: yes there was a commit to remove it just not pushed; please give me a second.

Comment (zero-24) [reply administrator/components/com_admin/postinstall/joomla40checks.php]: pushed. thanks.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 091489b54ba4b27bdc1979b5a51302c0cd53fa13<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27190">issues.joomla.org/tracker/joomla-cms/27190</a>.</sub>

**Comment by ot2sen** [comment]
With Joomla! 4 final release not being released until 2020 we should target a different minimum IMO.
Having a 'security fixes only' minimum from launch do not meet our usual standards of being ahead of the game. At least please make it PHP 7.3+ which has full support throughout 2020.
https://www.php.net/supported-versions.php

**Comment by mbabker** [comment]
How about don’t bump it again until the beta is tagged as this is now the fourth time it has changed in this iteration of 4.0.

Also, there is zero technical benefit to a 7.3 minimum over 7.2, a 7.3 minimum is purely for the “security support means branch is unsupported” crowd.  Personally I would jump 7.2 to 7.4 for minimum but I doubt usage statistics would support that one at this time.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 091489b54ba4b27bdc1979b5a51302c0cd53fa13<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27190">issues.joomla.org/tracker/joomla-cms/27190</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27190">issues.joomla.org/tracker/joomla-cms/27190</a>.</sub>

**Comment by wilsonge** [comment]
> meet our usual standards of being ahead of the game

In the past this wasn't exactly our standard for PHP versions - we deployed based on what was being deployed with linux lts operating systems. Which is the same standard that was used for 7.2.5 if you read the symfony issue tracker on them picking 7.2.5

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**